src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5883,6 +5893,9 @@ internal static class PSSnapInHelpers
                         var aliasList = new List<SessionStateAliasEntry>();
                         foreach (var alias in aliasAttribute.AliasNames)
                         {
+                            // We add aliases from custom modules with 'ScopedItemOptions.None'
+                            // because only a module consumer (not the module author) decides
+                            // whether the alias is read only and available in all scopes.
```

Comment:
My bad. This comment should be put here.
However, the comment may need a bit change, since `AliasAttribute` can be used in built-in module, not custom modules only. Maybe the following is a little better:
"Alias declared by AliasAttribute is set with the option 'ScopedItemOptions.None', because we believe a user of the cmdlet, instead of the author of it, should be the one to decide the option of the alias usage."